src/viperleed/gui/measure/classes/settings.py

```diff
@@ -92,7 +109,230 @@ class TooManyDefaultSettingsError(DefaultSettingsError):
+            Whether interpolations in the return value should be
+            expanded. If False, interpolations are expanded. Default
+            is False.
+        vars : dict, optional
+            If provided, the option is looked up in vars before
+            checking section. Default is None.
+        fallback : object, optional
+            If provided, fallback is the return value if getting the
+            option failed. Default is _UNSET.
+
+        Returns
+        -------
+        value : object
+            The read value from the loaded settings or the fallback.
+
+        Raises
+        ------
+        NoSectionError
+            If the settings section is missing.
+        NoOptionError
+            If the settings option is missing.
+        """
+        value = _UNSET
+        try:
+            value = super().get(section, option, raw=raw, vars=vars)
+        except (NoSectionError, NoOptionError):
+            value = self._get_from_alias(section, option, fallback,
+                                         raw=raw, vars=vars)
+            if value is _UNSET:
+                raise
+
+        if not value:
+            try:
+                # This fallback is not the fallback given by the user,
+                # but a fallback taken from the aliases.
+                value = self._fallback[section][option]
+            except KeyError:
+                pass
+            else:
+                self[section][option] = value
+
+        return value
+
+    def prepare_aliases(self, cls_name):
+        """Prepare aliases for the handled settings.
+
+        Parameters
+        ----------
+        cls_name : str
+            Name of the class to which the loaded settings belong.
+
+        Returns
+        -------
+        None.
+        """
+        self._aliases = {}
+        self._fallback = defaultdict(dict)
+        self._cls_name = cls_name
+        if not self._cls_name:
+            # No class name, therefore we cannot look up aliases.
+            return
+        self._load_aliases()
+
+    def _add_new_sections(self, new_sections):
+        """Add new settings sections.
+
+        Parameters
+        ----------
+        new_sections : list
+            A list of sections to add.
+
+        Returns
+        -------
+        None.
+        """
+        for section in new_sections:
+            if section in self.sections():
+                continue
+            self.add_section(section)
+
+    def _get_aliases(self, section, option):
+        """Get aliases for a section/option pair.
+
+        Parameters
+        ----------
+        section : str
+            The section key.
+        option : str
+            The option key.
+
+        Returns
+        -------
+        aliases : list
+            A list of section/option pairs.
+        """
+        try:
+            aliases = self._aliases[f'{section}/{option}']
+        except KeyError:
+            return []
+        return [alias.split('/') for alias in aliases]
```

Comment:
```suggestion
    def _iter_aliases(self, section, option):
        """Yield aliases for a section/option pair.

        Parameters
        ----------
        section : str
            The section key.
        option : str
            The option key.

        Yields
        ------
        alias : tuple
            Names of old section/option corresponding to the ones given.
        """
        try:
            aliases = self._aliases[f'{section}/{option}']
        except KeyError:
            return
        yield from (alias.split('/') for alias in aliases)
```

Since we only ever iterate over this (and break off at the first success) it may be worth making this an iterator. This prevents creation of all the section/option tuples every time.